staging/src/k8s.io/apiserver/Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/apiserver",
-	"GoVersion": "go1.9",
+	"GoVersion": "go1.10",
```

Comment:
Can we avoid changing the go version in this PR?

Reply:
I'd like to. Not sure what did it, probably the "update-godep-staging.sh". I don't even have go1.10 installed on my machine I think (it's probably done from within some sort of container).

Reply:
I guess as long as it is only generated files it's ok.